pmdk.spec

```
@@ -499,6 +430,26 @@ cp utils/pmdk.magic %{buildroot}%{_datadir}/pmdk/
 
 
 %changelog
+* Fri May 24 2024  Tomasz.Gromadzki <tomasz.gromadzki@intel.com> - 2.1.0-1
+  Update to release 2.1.0 w/o NDCTL support which:
```

Nitpick but not having a dash here is unprecedented for this file. I would recommend either adding it back or starting from the beginning of the line. Having two empty spaces at the beginning of the line just takes the worst from both solutions.

Note: Starting from the beginning of the line also means you probably should the list below unindent so all the dashes will be at the beginning of the respective lines.

```suggestion
- Update to release 2.1.0 w/o NDCTL support which:
```

---

Done